# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gfx-hal/src/pso/compute.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gfx-hal/src/pso/compute.rs b/third_party/rust/gfx-hal/src/pso/compute.rs
deleted file mode 100644
--- a/third_party/rust/gfx-hal/src/pso/compute.rs
+++ /dev/null
@@ -1,34 +0,0 @@
-//! Compute pipeline descriptor.
-
-use crate::{
-    pso::{BasePipeline, EntryPoint, PipelineCreationFlags},
-    Backend,
-};
-
-/// A description of the data needed to construct a compute pipeline.
-#[derive(Debug)]
-pub struct ComputePipelineDesc<'a, B: Backend> {
-    /// Pipeline label
-    pub label: Option<&'a str>,
-    /// The shader entry point that performs the computation.
-    pub shader: EntryPoint<'a, B>,
-    /// Pipeline layout.
-    pub layout: &'a B::PipelineLayout,
-    /// Any flags necessary for the pipeline creation.
-    pub flags: PipelineCreationFlags,
-    /// The parent pipeline to this one, if any.
-    pub parent: BasePipeline<'a, B::ComputePipeline>,
-}
-
-impl<'a, B: Backend> ComputePipelineDesc<'a, B> {
-    /// Create a new empty PSO descriptor.
-    pub fn new(shader: EntryPoint<'a, B>, layout: &'a B::PipelineLayout) -> Self {
-        ComputePipelineDesc {
-            label: None,
-            shader,
-            layout,
-            flags: PipelineCreationFlags::empty(),
-            parent: BasePipeline::None,
-        }
-    }
-}